### PR TITLE
Ulyanov2016instance Summary 2

### PR DESCRIPTION
**Pull request by kraja99**

### CODE REVIEW

File: _summaries/ulyanov2016instance_2.md [hunk 1]
```diff
@@ -1,19 +1,49 @@
...
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: kraja99
+score: x / 10
 ---
 
 TODO: Summarize the paper:
 * What is the core idea?
+  
+Ulyanov et. al. revist the task of image stylization. Previous works like Gatys et. al. (2016) have introduced methods for style transfer with good results but high computational cost.  Ulyanov et. al. (2016) introduced equivalent feed-forward generator networks, however these networks did not achieve results as good as those propsed in Gatys et. al. In this paper Ulyanov improves his generator architecture through the replacement of batch normalization layers with instance normalization layers and keep them at test time.
```
Comment (aabayomi) [inline _summaries/ulyanov2016instance_2.md:13]: I think the summary is good but i don't think it is not necessary to include the author's names. You may as well go straight to what the problem is and how it was solved.

File: _summaries/ulyanov2016instance_2.md [hunk 2]
```diff
@@ -1,19 +1,49 @@
...
 TODO: Summarize the paper:
 * What is the core idea?
+  
+Ulyanov et. al. revist the task of image stylization. Previous works like Gatys et. al. (2016) have introduced methods for style transfer with good results but high computational cost.  Ulyanov et. al. (2016) introduced equivalent feed-forward generator networks, however these networks did not achieve results as good as those propsed in Gatys et. al. In this paper Ulyanov improves his generator architecture through the replacement of batch normalization layers with instance normalization layers and keep them at test time. 
+
+![Image Stylization Ex.](ulyanov2016instance_2a.png)
+
 * How is it realized (technically)?
+
+The authors noted that the contrast of the input image should not be a factor in stylization and that contrast of the stylized image should be similar to the contrast of the original style image. As a result, the authors conclude that the network should discard contrast information from the content image.
```
Comment (aabayomi) [inline _summaries/ulyanov2016instance_2.md:19]: I think it is good start with an introduction why and how they came about the solution which is  instance normalization.Then go on explain the difference in algorithm between the previous method (batch normalization).

**Review by aabayomi — COMMENTED** [review]
Overall:
I think is it a good summary, just needs additional technical description of the papers main points.

Originality -> good 
Structure -> good.

File: _summaries/ulyanov2016instance_2.md [hunk 3]
```diff
@@ -1,19 +1,49 @@
...
+
+To combine instance-specific normalization and batch normalization, the authors propose instance normalization as follows:
+
+![Instance Normalization](ulyanov2016instance_2d.png)
+
+The main difference between the two being that instance normalization performs per image normalization.
+
 * How well does the paper perform?
+
+The authors performed some qualitative analysis on the substitution of batch normalization for instance normalization in both the Ulyanov et. al. (2016) and Johnson et. al. (2016) architectures, citing significant improvements over batch normalization.
```
[the remark this answers is not in the document]
Comment (aabayomi) [reply _summaries/ulyanov2016instance_2.md:39]: Also i think questions like training/testing performance should be answered.
